Stack clients: NoClassDefFoundError in Podman API call

In a custom agent (when trying to initialise a time series), the following exception has been observed:
```
: java.lang.NoClassDefFoundError: org/glassfish/jersey/innate/io/InputStreamWrapper
        at java.base/java.lang.ClassLoader.defineClass1(Native Method)
        at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1027)
        at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150)
        at java.base/jdk.internal.loader.BuiltinClassLoader.defineClass(BuiltinClassLoader.java:862)
        at java.base/jdk.internal.loader.BuiltinClassLoader.findClassOnClassPathOrNull(BuiltinClassLoader.java:760)
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClassOrNull(BuiltinClassLoader.java:681)
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:639)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
        at org.glassfish.jersey.apache.connector.ApacheConnectorProvider.getConnector(ApacheConnectorProvider.java:105)
        at org.glassfish.jersey.client.ClientConfig$State.initRuntime(ClientConfig.java:466)
        at org.glassfish.jersey.internal.util.collection.Values$LazyValueImpl.get(Values.java:317)
        at org.glassfish.jersey.client.ClientConfig.getRuntime(ClientConfig.java:829)
        at org.glassfish.jersey.client.ClientRequest.getClientRuntime(ClientRequest.java:176)
        at org.glassfish.jersey.client.JerseyWebTarget.onBuilder(JerseyWebTarget.java:371)
        at org.glassfish.jersey.client.JerseyWebTarget.request(JerseyWebTarget.java:199)
        at org.glassfish.jersey.client.JerseyWebTarget.request(JerseyWebTarget.java:38)
        at io.theworldavatar.swagger.podman.ApiClient.invokeAPI(ApiClient.java:654)
        at io.theworldavatar.swagger.podman.api.SecretsApi.secretCreateLibpodWithHttpInfo(SecretsApi.java:103)
        at io.theworldavatar.swagger.podman.api.SecretsApi.secretCreateLibpod(SecretsApi.java:53)
        at com.cmclinnovations.stack.clients.docker.PodmanClient.addSecret(PodmanClient.java:47)
        at com.cmclinnovations.stack.clients.docker.PodmanClient.addConfig(PodmanClient.java:60)
        at com.cmclinnovations.stack.clients.docker.DockerConfigHandler.writeEndpointConfig(DockerConfigHandler.java:55)
        at com.cmclinnovations.stack.services.ContainerService.writeEndpointConfig(ContainerService.java:193)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
        at com.cmclinnovations.stack.services.ContainerService.writeEndpointConfigs(ContainerService.java:116)
        at com.cmclinnovations.stack.services.DockerService.writeEndpointConfigs(DockerService.java:239)
        at com.cmclinnovations.stack.services.ServiceManager.initialiseService(ServiceManager.java:170)
        at com.cmclinnovations.stack.clients.timeseries.TimeSeriesRDBClient.configureOntop(TimeSeriesRDBClient.java:97)
        at com.cmclinnovations.stack.clients.timeseries.TimeSeriesRDBClient.bulkInitTimeSeriesTable(TimeSeriesRDBClient.java:74)
        at uk.ac.cam.cares.jps.base.timeseries.TimeSeriesClient.bulkInitTimeSeries(TimeSeriesClient.java:311)
        at uk.ac.cam.cares.jps.base.timeseries.TimeSeriesClient.bulkInitTimeSeries(TimeSeriesClient.java:1319)
        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
        at java.base/java.lang.reflect.Method.invoke(Method.java:580)
        at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:244)
        at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
        at py4j.Gateway.invoke(Gateway.java:282)
        at py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
        at py4j.commands.CallCommand.execute(CallCommand.java:79)
        at py4j.GatewayConnection.run(GatewayConnection.java:238)
        at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.lang.ClassNotFoundException: org.glassfish.jersey.innate.io.InputStreamWrapper
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
        ... 41 more
 ```
This problem occurs only when using Podman - the agent works fine when the stack is run using Docker.

--- a/stack-clients/docker-compose.yml
+++ b/stack-clients/docker-compose.yml
@@ -1,6 +1,6 @@
 services:
   stack-client:
-    image: ghcr.io/theworldavatar/stack-client${IMAGE_SUFFIX}:1.58.0
+    image: ghcr.io/theworldavatar/stack-client${IMAGE_SUFFIX}:1.58.1-dev-swagger-dependencies-SNAPSHOT
     secrets:
       - blazegraph_password
       - postgis_password

--- a/stack-clients/docker-compose.yml
+++ b/stack-clients/docker-compose.yml
@@ -1,6 +1,6 @@
 services:
   stack-client:
-    image: ghcr.io/theworldavatar/stack-client${IMAGE_SUFFIX}:1.58.1-dev-swagger-dependencies-SNAPSHOT
+    image: ghcr.io/theworldavatar/stack-client${IMAGE_SUFFIX}:1.58.1
     secrets:
       - blazegraph_password
       - postgis_password

--- a/stack-clients/pom.xml
+++ b/stack-clients/pom.xml
@@ -47,6 +47,19 @@
         </snapshotRepository>
     </distributionManagement>
 
+    <!-- Ensure consistent versions of Jersey dependencies generated dynamically by Swagger for Podman API -->
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.glassfish.jersey</groupId>
+                <artifactId>jersey-bom</artifactId>
+                <version>2.41</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
     <dependencies>
 
         <dependency>

--- a/stack-clients/pom.xml
+++ b/stack-clients/pom.xml
@@ -7,7 +7,7 @@
 
     <groupId>com.cmclinnovations</groupId>
     <artifactId>stack-clients</artifactId>
-    <version>1.58.1-dev-swagger-dependencies-SNAPSHOT</version>
+    <version>1.58.1</version>
 
     <name>Stack Clients</name>
     <url>https://theworldavatar.io</url>

--- a/stack-clients/pom.xml
+++ b/stack-clients/pom.xml
@@ -7,7 +7,7 @@
 
     <groupId>com.cmclinnovations</groupId>
     <artifactId>stack-clients</artifactId>
-    <version>1.58.0</version>
+    <version>1.58.1-dev-swagger-dependencies-SNAPSHOT</version>
 
     <name>Stack Clients</name>
     <url>https://theworldavatar.io</url>

--- a/stack-data-uploader/docker-compose.yml
+++ b/stack-data-uploader/docker-compose.yml
@@ -1,6 +1,6 @@
 services:
   stack-data-uploader:
-    image: ghcr.io/theworldavatar/stack-data-uploader${IMAGE_SUFFIX}:1.58.0
+    image: ghcr.io/theworldavatar/stack-data-uploader${IMAGE_SUFFIX}:1.58.1-dev-swagger-dependencies-SNAPSHOT
     secrets:
       - blazegraph_password
       - postgis_password

--- a/stack-data-uploader/docker-compose.yml
+++ b/stack-data-uploader/docker-compose.yml
@@ -1,6 +1,6 @@
 services:
   stack-data-uploader:
-    image: ghcr.io/theworldavatar/stack-data-uploader${IMAGE_SUFFIX}:1.58.1-dev-swagger-dependencies-SNAPSHOT
+    image: ghcr.io/theworldavatar/stack-data-uploader${IMAGE_SUFFIX}:1.58.1
     secrets:
       - blazegraph_password
       - postgis_password

--- a/stack-data-uploader/pom.xml
+++ b/stack-data-uploader/pom.xml
@@ -7,7 +7,7 @@
 
     <groupId>com.cmclinnovations</groupId>
     <artifactId>stack-data-uploader</artifactId>
-    <version>1.58.1-dev-swagger-dependencies-SNAPSHOT</version>
+    <version>1.58.1</version>
 
     <name>Stack Data Uploader</name>
     <url>https://theworldavatar.io</url>

--- a/stack-data-uploader/pom.xml
+++ b/stack-data-uploader/pom.xml
@@ -7,7 +7,7 @@
 
     <groupId>com.cmclinnovations</groupId>
     <artifactId>stack-data-uploader</artifactId>
-    <version>1.58.0</version>
+    <version>1.58.1-dev-swagger-dependencies-SNAPSHOT</version>
 
     <name>Stack Data Uploader</name>
     <url>https://theworldavatar.io</url>

--- a/stack-manager/docker-compose.yml
+++ b/stack-manager/docker-compose.yml
@@ -1,6 +1,6 @@
 services:
   stack-manager:
-    image: ghcr.io/theworldavatar/stack-manager${IMAGE_SUFFIX}:1.58.1-dev-swagger-dependencies-SNAPSHOT
+    image: ghcr.io/theworldavatar/stack-manager${IMAGE_SUFFIX}:1.58.1
     environment:
       EXTERNAL_PORT: "${EXTERNAL_PORT-3838}"
       STACK_BASE_DIR: "${STACK_BASE_DIR}"

--- a/stack-manager/docker-compose.yml
+++ b/stack-manager/docker-compose.yml
@@ -1,6 +1,6 @@
 services:
   stack-manager:
-    image: ghcr.io/theworldavatar/stack-manager${IMAGE_SUFFIX}:1.58.0
+    image: ghcr.io/theworldavatar/stack-manager${IMAGE_SUFFIX}:1.58.1-dev-swagger-dependencies-SNAPSHOT
     environment:
       EXTERNAL_PORT: "${EXTERNAL_PORT-3838}"
       STACK_BASE_DIR: "${STACK_BASE_DIR}"

--- a/stack-manager/pom.xml
+++ b/stack-manager/pom.xml
@@ -7,7 +7,7 @@
 
     <groupId>com.cmclinnovations</groupId>
     <artifactId>stack-manager</artifactId>
-    <version>1.58.0</version>
+    <version>1.58.1-dev-swagger-dependencies-SNAPSHOT</version>
 
     <name>Stack Manager</name>
     <url>https://theworldavatar.io</url>

--- a/stack-manager/pom.xml
+++ b/stack-manager/pom.xml
@@ -7,7 +7,7 @@
 
     <groupId>com.cmclinnovations</groupId>
     <artifactId>stack-manager</artifactId>
-    <version>1.58.1-dev-swagger-dependencies-SNAPSHOT</version>
+    <version>1.58.1</version>
 
     <name>Stack Manager</name>
     <url>https://theworldavatar.io</url>

